tests/test_keycloak_admin.py

```diff
@@ -204,6 +204,15 @@ def test_realms(admin: KeycloakAdmin) -> None:
     assert "master" in realm_names, realm_names
     assert "test" in realm_names, realm_names
 
+    # Get and update users profile
+    user_profile = admin.get_realm_users_profile()
+    assert "attributes" in user_profile
+    assert "groups" in user_profile
+
+    res = admin.update_realm_users_profile(user_profile)
```

Comment:
Could we also test updating some attributes/groups? Just to be sure that we're not only hitting an edge case here as the profile is unchanged.

Reply:
Sure!